src/applications/edu-benefits/10203/containers/ConfirmationPage.jsx

```diff
@@ -101,25 +101,20 @@ class ConfirmationPage extends React.Component {
           <h4 className="confirmation-guidance-heading">
             What happens after I apply?
           </h4>
-          <p className="confirmation-guidance-message">
-            We usually decide on applications within 30 days.
-          </p>
-          <p className="confirmation-guidance-message">
+          <p>We usually decide on applications within 30 days.</p>
+
+          <p>
             You’ll get a Certificate of Eligibility (COE), or award letter, in
             the mail if we’ve approved your application. Bring this to the VA
             certifying official at your school.
           </p>
-          <p className="confirmation-guidance-message">
+          <p>
```

Comment:
what happened to the usages of `confirmation-guidance-message`?

Reply:
I can add them back in if desired, but got an informal UX approval without them. Currently there is only one usage of that for this here:

```
<p className="confirmation-guidance-message">
            If you have questions, call 1-888-GI-BILL-1 (
            <a href="tel:+18884424551">1-888-442-4551</a>
            ), Monday &#8211; Friday, 8:00 a.m. &#8211; 7:00 p.m. ET.
</p>
```

Reply:
What did removing the css class do for the confirmation page?

Reply:
Not much in the way of styling, it looks as though they may have been added as a way to select those components in a unit test as I see those are failing after being removed here.